src/screens/Notification.js

```diff
@@ -193,6 +214,58 @@ return (
           </PropertyValue>
         </Property>
 
+        <Property name="notification.global.container">
+          <Description>
+            Any valid Box prop for the Global Notification container.
+          </Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`
+  {
+  pad: {
+    horizontal: 'large',
+    vertical: 'xsmall',
+  },
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+        <Property name="notification.direction">
+          <Description>
+            The layout direction of the title and message.
+          </Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`
+  {
+  pad: {
+    horizontal: 'large',
+    vertical: 'xsmall',
+  },
+                `}
+            </Example>
+          </PropertyValue>
```

Comment:
Is this correct? Was expecting 'row' and 'column'

Reply:
Fixed.